### DISCUSSION TITLE
Navigation file explorer with h, j, yank, etc

### DISCUSSION THREAD

**Discussion by fazulk** [JetBrains/ideavim#889] [General]
New to webstorm, coming from vscode looking for something that is a little more polished. One of the features I use in vscode with neovim is being able to use `h, j, yank, rename, p` etc inside file explorer, and also navigating the project files modal with `h` and `j`. Is something like possible in webstorm?

**Comment by OptimusCrime** [comment]
I think that you're looking for the NERDtree extension to IdeaVim. See documentation here: https://github.com/JetBrains/ideavim/wiki/NERDTree-support

**Comment by citizenmatt** [comment]
As already noted, the NERDTree extension can help with the Project view. However, it is only available here, since it requires specific handling for each tool window. Since you mention you're new to WebStorm, it's worth pointing out that practically everywhere you see a list or tree, you can just start typing to search inside it. It's not Vim-like, but it's still a very effective way to navigate inside lists.

**Reply by MorKaragh** [reply]
For some reason creating new files or new directories does not work in Idea Community, instead typing "n" or "N" just starts the search.

**Reply by AlexPl292** [reply]
Hi there. This feature is not originally presented in NERDTree. However, it was proposed by the user some time ago and it will be available on the next IdeaVim release: https://github.com/JetBrains/ideavim/pull/912
